src/tests/unit/tests/DetailsView/components/save-assessment-dialog.test.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are there fewer tests than when these tests lived in `SaveAssessmentButton`?

Reply:
Hi @madalynrose  as we moved out SaveAssessmentDialog from save-assessment-button, hence dialog related tests and save assessment button tests are separated.

Reply:
yes but it appears that there are fewer total now. I compared across both files.

Reply:
Hi @madalynrose added now the missing tests, please re-review